DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureInfo.java

```diff
@@ -0,0 +1,6 @@
+package com.teamagam.dailyselfie;
+
+class PictureInfo {
+    String mPath;
```

Comment:
We actually don't have a coding convention for public fields naming, but the m prefix is usually used to denote a private field member.